src/autotrash/app.py

```diff
@@ -322,6 +322,8 @@ def main():
                 if deleted_target or options.stat:
                     deleted_size += file_info['size']
                     deleted_files += 1
+            elif options.verbose:
+                logging.log(VERBOSE, 'keep %s', real_file_name(file_info['trash_info']))
```

Comment:
All messages currently start with a capital letter, I am going to accept the PR and change it after the merge. Just so you know for future patches.